substrate/frame/nomination-pools/src/migration.rs

```diff
@@ -37,6 +37,94 @@ pub mod versioned_migrations {
 	>;
 }
 
+/// This migration accumulates and initializes the [`TotalValueLocked`] for all pools.
+///
+/// WARNING: This migration works under the assumption that the [`BondedPools`] cannot be inflated
+/// arbitrarily. Otherwise this migration could fail due to too high weight.
+pub mod v7 {
```

Comment:
This can be made private and `VersionCheckedMigrateV6ToV7 ` moved to the public module `versioned_migrations`.

More context [here](https://github.com/paritytech/polkadot-sdk/blob/222ed3acba8b27d571a2a85d1678cf481f17d042/substrate/frame/examples/storage-migrations/src/migrations/v1.rs#L39-L157).

Reply:
I see, thanks for the finishing touch 🙏